common/lib/capa/capa/capa_problem.py

```diff
@@ -62,6 +74,9 @@
 # main class for this module
 
 
+DEFAULT_QUESTION_TEXT = _("You must specify meaningful question text.")
```

Comment:
For now we're going with text only. @muhammad-ammar Can you use this suggested text ?